counter-sink/src/main/resources/application.yml

```diff
@@ -0,0 +1,7 @@
+server:
+  port: 8081
+spring:
+  cloud:
+    stream:
+      bindings:
+        input: ticktock
```

Comment:
Yeah, I wasn't suggesting that the discussion was meaningless, just want to move it elsewhere (and reference it), as it doesn't prevent the current PR from being merged. :)